libr/include/r_search.h

```diff
@@ -34,8 +34,6 @@ typedef struct r_search_keyword_t {
 	ut8 *bin_binmask;
 	ut32 keyword_length;
 	ut32 binmask_length;
-	ut32 idx[R_SEARCH_DISTANCE_MAX]; // searching purposes
```

Comment:
we can discuss this thing in a separate issue/pr, better change only 1 thing on each PR, this way it's easier to review